src/Unicorn.Users/Loader/UserLoader.cs

```diff
@@ -81,27 +72,18 @@ protected virtual void DeserializeUser(SyncUserFile serializedUser)
 			bool addedUser;
 			var existingUser = GetOrCreateUser(serializedUser, out addedUser);
 
-			MembershipUser updatedUser = new MembershipUser(
-				existingUser.ProviderName,
-				syncUser.UserName,
-				existingUser.ProviderUserKey,
-				syncUser.Email,
-				null,
-				syncUser.Comment,
-				syncUser.IsApproved,
-				existingUser.IsLockedOut,
-				syncUser.CreationDate,
-				existingUser.LastLoginDate,
-				existingUser.LastActivityDate,
-				addedUser ? DateTime.UtcNow : existingUser.LastPasswordChangedDate,
-				existingUser.LastLockoutDate);
+			var updatedUser = new MembershipUser(existingUser.ProviderName, syncUser.UserName, existingUser.ProviderUserKey, syncUser.Email, null, syncUser.Comment, syncUser.IsApproved, existingUser.IsLockedOut, syncUser.CreationDate, existingUser.LastLoginDate, existingUser.LastActivityDate, addedUser ? DateTime.UtcNow : existingUser.LastPasswordChangedDate, existingUser.LastLockoutDate);
 
 			if (!addedUser)
 			{
-				if (!existingUser.Email.Equals(updatedUser.Email)) changes.Add(new UserUpdate("Email", existingUser.Email, updatedUser.Email));
-				if (!existingUser.Comment.Equals(updatedUser.Comment)) changes.Add(new UserUpdate("Comment", existingUser.Comment, updatedUser.Comment));
-				if (!existingUser.IsApproved.Equals(updatedUser.IsApproved)) changes.Add(new UserUpdate("IsApproved", existingUser.IsApproved.ToString(), updatedUser.IsApproved.ToString()));
-				if (!existingUser.CreationDate.Equals(updatedUser.CreationDate.ToUniversalTime())) changes.Add(new UserUpdate("CreationDate", existingUser.CreationDate.ToString(CultureInfo.InvariantCulture), updatedUser.CreationDate.ToUniversalTime().ToString(CultureInfo.InvariantCulture) + " (UTC)"));
+				if (existingUser.Email != null && !existingUser.Email.Equals(updatedUser.Email))
+					changes.Add(new UserUpdate("Email", existingUser.Email, updatedUser.Email));
+				if (existingUser.Comment != null && !existingUser.Comment.Equals(updatedUser.Comment))
+					changes.Add(new UserUpdate("Comment", existingUser.Comment, updatedUser.Comment));
+				if (!existingUser.IsApproved.Equals(updatedUser.IsApproved))
+					changes.Add(new UserUpdate("IsApproved", existingUser.IsApproved.ToString(), updatedUser.IsApproved.ToString()));
+				if (!existingUser.CreationDate.Equals(updatedUser.CreationDate.ToUniversalTime()))
+					changes.Add(new UserUpdate("CreationDate", existingUser.CreationDate.ToString(CultureInfo.InvariantCulture), updatedUser.CreationDate.ToUniversalTime().ToString(CultureInfo.InvariantCulture) + " (UTC)"));

```

Comment:
For multi-line `if` statements please add curly braces (e.g.

```
if(foo)
{
 // do stuff
}
```

Or else return the formatting to how it was.